skopt/optimizer/base.py

```diff
@@ -162,6 +162,10 @@ def base_minimize(func, dimensions, base_estimator,
         "n_jobs": n_jobs}
     acq_func_kwargs = {"xi": xi, "kappa": kappa}
 
+    if acq_optimizer == "auto":
+        warnings.warn("The 'auto' option for the acq_optimizer will be "
+                      "removed in 0.4.")
```

Comment:
Shouldn't we remove this? The next release will be v0.4 so we can't remove the option?